functions/src/index.ts

```diff
@@ -77,8 +85,14 @@ async function handleAddDocument(
  */
 async function handleDeleteDocument(documentId: string): Promise<void> {
   try {
-    await index.deleteDocument(documentId)
     logs.deleteDocument(documentId)
+    if (validateDocumentId(documentId)) {
+      await index.deleteDocument(documentId)
+    } else {
+      logger.error(
+        `Could not create document with id: ${documentId}. The document id can only contain case-insensitive alphanumeric characters (abcDEF), hyphens (-) or underscores(_).`
```

Comment:
Shouldn't it be something like: `Could not delete document` or `Document can't exist with id: ...`